index.html

```diff
@@ -25,8 +25,8 @@ <h1>About Sydnie</h1>
             <section class="about">
                 <h2>About Me</h2>
                 <figure>
-                    <img src="Image/img.jpg" alt="Image of a laptop">
-                    <figcaption>Here is a image of laptop.</figcaption>
+                    <img src="assets/Headshot21.jpg" alt="Image of a laptop">
```

Comment:
Super close here! Images can definitely be difficult to work with. In this case, you'll want to use _relative_ pathing to get to your assets. Relative pathing looks like this `"./assets/Headshot21.jpg"` where the `./` indicates the number of directories you need to move (since the assets folder is in the same root directory as index.html, you only need 1 `./`). 

You also need to make sure you add the image you want to use to your assets folder so that your app will have access to view it.

Reply:
Great job using semantic HTML with the `<figure>` tag, by the way! 💯